WatchOS/MainApp.WatchAppExtension/MainApp.WatchAppExtension.csproj

```diff
@@ -48,6 +48,7 @@
     <IOSDebugOverWiFi>false</IOSDebugOverWiFi>
     <MtouchVerbosity></MtouchVerbosity>
     <CodesignProvision>WatchKit Catalog Watch App Extension</CodesignProvision>
+    <MtouchExtraArgs>--abi=armv7k+llvm,arm64_32+llvm</MtouchExtraArgs>

```

Comment:
Why not do `<MtouchArch>ARMv7k, ARM64_32</MtouchArg>` further above?

Reply:
don't we need LLVM to be enabled (bitcode) to submit ?

Reply:
Yes, you need to add `<MtouchUseLlvm>true</MtouchUseLlvm>` to enable LLVM if it's not already there (but it looks like it is).

Reply:
was you comment about the `Debug` config then ? or am I missing something obvious ?!?

Reply:
The Release configuration already has llvm enabled: https://github.com/xamarin/SubmissionSamples/blob/ca4ca8282a64769834667bd1fde777ae3593ceba/WatchOS/MainApp.WatchAppExtension/MainApp.WatchAppExtension.csproj#L41

so changing this line: https://github.com/xamarin/SubmissionSamples/blob/ca4ca8282a64769834667bd1fde777ae3593ceba/WatchOS/MainApp.WatchAppExtension/MainApp.WatchAppExtension.csproj#L46

to:

```csharp
<MtouchArch>ARMv7k, ARM64_32</MtouchArg>
```

should accomplish the same thing your change did (and it would be the standard way of doing it).

Reply:
ah, I see. I did not want to fight VS or get failures before `mtouch` when running older versions (but I have not really tried if it was an issue or not, I just expected some)

Reply:
Either your way or my way will fail to build on older XIs.

With my way VS will revert the changes if you change the csproj for some other reason. OTOH my way will make more sense when you look at the csproj again in a couple of years...

Reply:
yes, but I wanted to failure to be clear (and I knew `mtouch` error would be). I'll try the .csproj in my next submissions and, if clear, will PR this separately.